src/metrics/config.ml

```diff
@@ -0,0 +1,2 @@
+let namespace = "Deku"
+let subsystem = "node"
```

Comment:
I think the subsystem maybe should be per module, so `block`should be one, but this is something we can tweak as we go along